skrf/network.py

```diff
@@ -2691,7 +2692,7 @@ def extrapolate_to_dc(self, points: int = None, dc_sparam: NumberLike = None, ki
         dc_sparam : class:`npy.ndarray` or None
             NxN S-parameters matrix at 0 Hz.
             If None S-parameters at 0 Hz are determined by linear extrapolation.
-        kind : str or int
+        kind : str or int, default is 'rational'
```

Comment:
Maybe also write here, that this default is going to be changed to 'cubic' in the future.

Reply:
done